modules/oc-mirror-support.adoc

```diff
@@ -9,6 +9,11 @@
 
 The oc-mirror plugin supports mirroring {product-title} payload images and Operator catalogs for {product-title} versions 4.10 and later.
 
+[NOTE]
+====
+On `ppc64le` and `s390x` architectures the oc-mirror plugin is only supported for {product-title} versions 4.14 and later.
```

Comment:
Hi Silke! This note also applies to the ARM arch, can you add `aarch64` arch, thank you
```suggestion
On `aarch64`, `ppc64le` and `s390x` architectures the oc-mirror plugin is only supported for {product-title} versions 4.14 and later.
```